### ISSUE TITLE
Please add new option "Run NCI tasks when on battery" to Android app

### ISSUE THREAD

**Issue by sirzooro** [BOINC/boinc#1905]
As in title. NCI tasks consumes little CPU, so user may want to run them when device is on battery too.

**Comment by ChristianBeer** [comment]
This does not exclusively apply to Android and is also relevant for Linux on ARM where you usually have a battery. Relates to general preference changes as suggested in #41

**Comment by davidpanderson** [comment]
Running any BOINC app causes a noticeable increase in battery drain rate, because of the polling loops in client and API.  So running NCI apps while on battery shouldn't be the default.  It could be an option.  What projects have NCI Android apps?

**Comment by sirzooro** [comment]
Yes, this should be disabled by default. I know about following NCI projects with Android apps: WUProp, all 6 Quake Catcher Networks projects, and GoofyxGrid (currently not working, project is moving to Grid Computing Center server).

**Comment by krzyszp** [comment]
Also Radioactive@Home running NCI app and it should by not stopped...

**Comment by sirzooro** [comment]
After some time I come to conclusion that this new option could be more general - namely "Run NCI tasks when suspended". BOINC Clients on Windows and Linux would benefit even more from it, as they usually do not run on battery.
